docs/howto/deployment/checklist.txt

```diff
@@ -187,6 +184,16 @@ Performance optimizations
 Setting :setting:`DEBUG = False <DEBUG>` disables several features that are
 only useful in development. In addition, you can tune the following settings.
 
+Sessions
+--------
+
+Consider using :ref:`cached sessions <cached-sessions-backend>` to improve
+performance.
+
+If using database-backed sessions, regularly
+:ref:`clear old sessions <clearing-the-session-store>` to avoid storing
+unnecessary data.
```

Comment:
You can split `ref`:
```
If using database-backed sessions, regularly :ref:`clear old sessions
<clearing-the-session-store>` to avoid storing unnecessary data.
```

Reply:
Yes sorry, it always just worries me since it breaks syntax highlighting and I know in some text formats such split links don't work

Reply:
Done in 47587b0a7e441aa6b3a088075b232b4aa79c7c51